### DISCUSSION TITLE
adding border inside image not around

### DISCUSSION THREAD

**Discussion by zeroheure** [ImageMagick/ImageMagick#3216] [Help]
Hello
I want to add a border on images, not around the current border, but inside it. Is there a simple parameter of a CLI option to do that? 
Thanks
Xavier

**Comment by snibgo** [comment]
You could "-shave" the edges before "-border". See https://imagemagick.org/script/command-line-options.php#shave

**Reply by zeroheure** [reply]
Thanks.
